src/operator/nn/dnnl/dnnl_pooling-inl.h

```diff
@@ -28,10 +28,13 @@
 
 #include <dnnl.hpp>
 #include <utility>
+#include <vector>
 
 #include "../pooling-inl.h"
 #include "./dnnl_base-inl.h"
 
+#define DIV_ROUND_UP(a, b) (((a) + (b - 1)) / b)
```

Comment:
Does a have to be in brackets?

Reply:
Not necessarily, changed for better readability.